packages/codelabs/src/basics/web-components.md

```diff
@@ -358,15 +358,15 @@ LitElement offers a lot more features than just rendering templates, check out o
 
 ## Shadow DOM
 
-The last important web component API we will look into is shadow DOM. Traditionally, HTML and CSS have always been global. This scales pretty badly, we need to make sure that ids are unique on the page and CSS selector can get pretty complex. This is why many front-end frameworks offer some form of encapsulation. With shadow DOM, this capability is now built into the browser.
+The last important API in Web Components we will look into is Shadow DOM. Traditionally, the context of HTML and CSS have always been global. This scales pretty badly, because we then constantly need to make sure that the id's of all the elements are unique and often CSS selectors can get pretty complex. This is why many front-end frameworks offer some form of encapsulation. Web Components provides us with this capability using `Shadow DOM`, this capability is now also built into the browser.
```

Comment:
```suggestion
The last important Web Components API we will look into is Shadow DOM. Traditionally, the context of HTML and CSS have always been global. This scales pretty badly, because we constantly need to make sure that the id's of all the elements are unique and often CSS selectors can get pretty complex. This is why many front-end frameworks offer some form of encapsulation. Web Components provides us with this capability using `Shadow DOM`, this capability is now built into the browser.
```